services/workflows-service/src/workflow/dtos/get-workflows-runtime-output.dto.ts

```diff
@@ -18,3 +19,5 @@ export class GetWorkflowsRuntimeOutputDto {
   @ApiProperty({ type: () => Pagination })
   meta!: Pagination;
 }
+
+export const GetWorkflowPluginOutput = WorkflowExtensionSchema;
```

Comment:
Why is this const needed? We can just use `WorkflowExtensionsSchema`

Reply:
Just to make the schema visible on dto file